src/encoded/static/components/regulome_search.js

```diff
@@ -258,20 +258,36 @@ ExampleEntry.propTypes = {
     handleExample: PropTypes.func.isRequired,
 };
 
-const AssemblySelector = props => (
-    <div className="assembly-switch">
-        <div className={`switch-label ${props.selection === 'GRCh38' ? 'active' : ''}`}>GRCh38</div>
-        <label className="switch" id="assembly-switch">
-            <input
-                type="checkbox"
-                onChange={() => { props.onSelection(props.selection); }}
-                checked={props.selection === 'hg19'}
-            />
-            <span className="slider round" />
-        </label>
-        <div className={`switch-label ${props.selection === 'hg19' ? 'active' : ''}`}>hg19</div>
-    </div>
-);
+const AssemblySelector = (props) => {
+    // need to display a focus style on "switch-label" because the input default checkbox is hidden and focus is therefore hidden
+    const [focus, toggleFocus] = React.useState(false);
+
+    return (
+        <div className="assembly-switch">
+            <div className={`switch-label ${props.selection === 'GRCh38' ? 'active' : ''}`}>GRCh38</div>
+            <label
+                className={`switch ${focus ? 'focused' : ''}`}
+                htmlFor="assembly-switch"
+                aria-labelledby={`Assembly is ${props.selection}. Toggle to switch to ${props.selection === 'GRCh38' ? 'hg19' : 'GRCh38'}`}
```

Comment:
I think this should use `aria-label`. [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) only holds the id of another element.